vars/RedHat.yml

```diff
@@ -2,5 +2,12 @@
 # File: RedHat.yml - Red Hat OS variables for Consul
 
 consul_os_packages:
-  - libselinux-python
+  - "{% if ( ansible_distribution  == 'Fedora' and ansible_distribution_version is version('31', '<') ) or \
```

Comment:
This logic added because if OS have python 3 by default it require `python3-libselinux` instead of `libselinux-python`